web/src/index.ts

```diff
@@ -115,7 +115,7 @@ const sendSocketCommand = wat => {
   if (wat.cmd == 'connect') {
     // console.log("connecting!");
     // let socket = new WebSocket(wat.address, wat.protocol);
-    let socket = new WebSocket(wat.address);
+    let socket = new WebSocket(wat.address, "asdf");
```

Comment:
The frontend is producing the following error when connecting over the websocket:

```
Error during WebSocket handshake: Sent non-empty 'Sec-WebSocket-Protocol' header but no response was received
```

It looks like the backend isn't expecting the `Sec-WebSocket-Protocol` header. The frontend connects appropriately when the `"asdf"` param is removed.

Reply:
Yeah that's a big mistake on my part. That should have been removed before making the PR but it was overlooked. Remnants of when we were playing around trying to find the correct place to put the JWT.

I imagine that error will go away after this correction is made.